index.d.ts

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+export type Day = {
+  year: number;
+  month: number;
+  day: number;
+};
+
+export type Range<T> = { from?: T; to?: T };
```

Comment:
Why Range is generic? I see its Day everywhere.

Reply:
I agree. There is no other use case for it.